controllers/commentController.js

```diff
@@ -13,18 +13,27 @@ const profileAtrributes = ['username', 'bio', 'image'];
 exports.postComment = async (req, res) => {
   try {
     const values = utility.trimValues(req.body);
```

Comment:
I think you can add a brief comment to this section in other to describe what this code does.

Reply:
I see, thanks.